src/components/DocumentationTopic/PrimaryContent/Mention.vue

```diff
@@ -0,0 +1,90 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="link-block">
+    <Reference
+      :url="url"
+      :role="role"
+      :kind="kind"
+      class="link"
+    >
+      <TopicLinkBlockIcon class="mention-icon" :role="role" />
+      <WordBreak tag="span">{{ title }}</WordBreak>
```

Comment:
It's probably safe to leave out the `WordBreak` component since these mentioned in links will always be article titles. The word break helper is more useful when something can contain a long single word, like some symbol names.

Reply:
Ah okay, I thought this was affecting word wrapping generally.